### DISCUSSION TITLE
Identify annotation in "annotations" user interface by its annotation id

### DISCUSSION THREAD

**Discussion by GMDuncan** [biigle/core#412] [Ideas]
We occasionally have quality control flags presented to us from downstream analyses and general QC checks using the csv reporting outputs, for example where the area for an annotation is spurious.

At this stage the annotation is (justly) only directly identifiable by its "id" value. It would be useful if there were a way for an annotator, using the application user interface, to quickly identify the annotation in question so that they confirm the spurious annotation within the system itself.

Potentially solvable by exposing the attribute id as text in the annotation tab like below, for a visual search:
![image](https://user-images.githubusercontent.com/11410721/149792479-8234883a-6474-4e9a-ae05-ed9a3c5715aa.png)
(I can see this id value is already requested from the database to form the "data-annotation-id" attribute in each list item)
Or by allowing annotation id as an option in the annotation tab's filter?

(New functionality rather than a bug)

**Comment by mzur** [comment]
Would using the `https://biigle.de/annotations/[id]` endpoint be a viable alternative? This redirects to the correct image and highlights the annotation in the user interface. You could generate the links in your downstream pipeline. Example: https://biigle.de/annotations/12305719

The endpoint is undocumented because it's not part of the REST API. Maybe we should update the documentation to include all HTTP endpoints sometime.

**Reply by GMDuncan** [reply]
Very nice and yes that would I think solve the requirement :)

Apologies for missing this!

**Reply by mzur** [reply]
No need to apologise, it's not documented after all :wink:
